### PR TITLE
Add series and webcomics to ‘about’ linked document field

### PR DESCRIPTION
**Pull request by gestchild**
## What does this change?

Following [a request from Lalita](https://wellcome.slack.com/archives/CUA669WHH/p1727260676138799) this adds the ability to add  Stories serials and webcomics to the 'Related stories' section of an exhibition page.

## How to test

I've deployed this change to prismic staging, so you can add these types to the page and preview the page locally.

## How can we measure success?

Content editors can add stories serials and webcomics to the 'Related stories' section of an exhibition page and they will render on the page.

## Have we considered potential risks?

None I can think of

### CODE REVIEW

**Comment by rcantin-w** [comment]
Is it worth adding `webcomic-series` even if it's just Body Squabbles?

**Review by rcantin-w — COMMENTED** [review]
The `.default` being added everywhere I'd like to understand, but I don't think is a blocker. @davidpmccormick had it last week in another PR as well and it didn't seem to affect anything...

File: common/customtypes/exhibitions/index.json [hunk 1]
```diff
@@ -183,7 +183,7 @@
               "config": {
                 "label": "Article",
```
Comment (rcantin-w) [inline common/customtypes/exhibitions/index.json:184]: Should/could the label be modified?

**Review by rcantin-w — APPROVED** [review]
Added [here](https://wellcomecollection-stage.prismic.io/builder/pages/ZpE1ghAAACQAgRSg?s=unclassified&section=About+this+exhibition)  and it looks good

Screenshot 2024-09-25 at 17 21 11

The main comment I'd care for more that I'd left is the label of the field on the CMS-side. Otherwise, all good to go!

**Comment by gestchild** [comment]
> Is it worth adding `webcomic-series` even if it's just Body Squabbles?

Weird, and if you then add the body squabbles series to the list, the 'Related stories' section doesn't render. Without it it's fine. Given that, and since it's not been asked for, I think I'll leave it off for now.

**Comment by gestchild** [comment]
> The `.default` being added everywhere I'd like to understand, but I don't think is a blocker. @davidpmccormick had it last week in another PR as well and it didn't seem to affect anything...

It's added by slice machine, but looks to only be in comments, so I think it's fine

File: common/customtypes/exhibitions/index.json (hunk 1 above)
Comment (gestchild) [reply common/customtypes/exhibitions/index.json]: I've changed to 'Story or Story series' as webcomics are a type of story now - does that sound right to you?

Comment (rcantin-w) [reply common/customtypes/exhibitions/index.json]: Yeah I think that works! It's mostly for editors so it's good to mention series too!

**Pull request merged**